linux/shim/init.go

```diff
@@ -120,6 +122,8 @@ func (s *Service) newInitProcess(context context.Context, r *shimapi.CreateTaskR
 		workDir:   s.config.WorkDir,
 		status:    0,
 		waitBlock: make(chan struct{}),
+		IoUID:     int(options.IoUid),
```

Comment:
nit: should we changes these to `IOGid` `IOUid`?

Reply:
linter told me to have UID/GID. I guess technically it should be `IOGUID` but meh

Reply:
oh Go